Curtains/Curtains.Application/AutoMapping/AutoMap.cs

```diff
@@ -13,7 +13,22 @@ public class AutoMap : Profile
     {
         public AutoMap()
         {
-            CreateMap<ReviewModel, ReviewDTO>().ReverseMap();       
+            CreateMap<ReviewModel, ReviewDTO>().ReverseMap();
+            CreateMap<ColorModel, ColorDTO>().ReverseMap();
+            CreateMap<ConsistencyModel, ConsistencyDTO>().ReverseMap();
+            CreateMap<CurtainsKindModel, CurtainsKindDTO>().ReverseMap();
+            CreateMap<CurtainsTypeModel, CurtainsTypeDTO>().ReverseMap();
```

Comment:
i think we need to split this class into several smaller

Reply:
In what way do you want to split them?

Reply:
we do not want to have many lines in one constructor :)

Reply:
so we can split this one into for example colorprofile, curtainsprofile etc

Reply:
Done!